src/i18n/locale/en.json

```diff
@@ -62,8 +62,8 @@
           },
           "label": {
             "action": "Download",
-            "error_one": "Error adding download",
-            "error_other": "Error adding downloads",
+            "error_one": "Could not add the download",
```

Comment:
download is a proper noun

Reply:
It is just a noun, for which there is no capitalisation (like in German).